src/scenes/statsScene.js

```diff
@@ -162,6 +166,7 @@ export class StatsScene extends Phaser.Scene {
 
             let name = "";
             let text = "";
+            let disaplayPercent = percent;
 
```

Comment:
Is this variable necessary? I don't understand its purpose
(small typo: display I guess)

Reply:
I use it to set the size of the percentage bar (which is sometimes 100 - percent)